include/cute/arch/copy_xe.hpp

```diff
@@ -98,8 +129,27 @@ struct XE_2D_U16x8x16x1x1_LD_N
       CUTE_INVALID_CONTROL_PATH("Trying to use block loads on non-PVC hardware");
     #endif
   }
+
+  struct PREFETCH {
+      template <class T>
+      CUTE_HOST_DEVICE static void copy(const void *baseoffset, int width,
+              int height, int pitch, intel::coord_t coord) {
+#if defined(SYCL_INTEL_TARGET)
+      static_assert(sizeof(T) == 2, "Expected T to have size 2");
+      __builtin_IB_subgroup_block_read_prefetch_u16_m8k16v1(
+              (long)baseoffset, width - 1, height - 1, pitch - 1, coord,
+              CacheControl::kL1C_L3C);
+#else
+      CUTE_INVALID_CONTROL_PATH(
+              "Trying to use block prefetch on non-PVC hardware");
+#endif
+        }
+    };
 };
 
+/// @brief This function loads data from 2D memory surface.
+/// Loads an array of rectangular regions coord(X,Y)..coord(X+W,Y+H) from global memory into registers.
+/// Loads 1x1 memory blocks, and each block size is 8x16x32bits
```

Comment:
why do we use different ordering of dimensions for the comment and the name of the struct?